packaging/obs/prometheus-hanadb_exporter/prometheus-hanadb_exporter.spec

```diff
@@ -23,7 +24,9 @@
 %define _prefix /usr
 %define oldsyscondir /etc
 %define _sysconfdir %{_prefix}/etc
+%define pythons python3
```

Comment:
This `pythons` define is not used anywhere.
Is it some new macro to define the used python versions in the next macros like `python_build`?

Reply:
> This pythons define is not used anywhere.
> Is it some new macro to define the used python versions in the next macros like python_build?

As far as I understand it, it tells the macros to only build for python3.
I grabbed it from here:
https://en.opensuse.org/openSUSE:Packaging_Python#Python_3_.28Leap_and_Tumbleweed.29

If I don't add it, python2 and python3 is being used to build this and this is not what we want in this case here, right?

Reply:
Yes, we only want `python3`.
Go ahead them!